planner_cspace/src/planner_3d.cpp

```diff
@@ -907,6 +905,41 @@ class Planner3dNode
         0.0,
         "second"));
     publishDebug();
+    return;
+  }
+
+  void cbNoMapUpdateTimer(const ros::TimerEvent& e)
+  {
+    planPath(e.current_real);
+    no_map_update_timer_ =
+        nh_.createTimer(costmap_watchdog_, &Planner3dNode::cbNoMapUpdateTimer, this, true);
+  }
+  void cbMapUpdate(const costmap_cspace_msgs::CSpace3DUpdate::ConstPtr msg)
+  {
+    if (!has_map_)
+      return;
+    ROS_DEBUG("Map updated");
+    if (trigger_plan_by_costmap_update_)
+    {
+      no_map_update_timer_.stop();
+      updateStart();
+      if (jump_.detectJump())
+      {
+        bbf_costmap_.clear();
+        // Robot pose jumped.
+      }
```

Comment:
As jump detector has constant threshold, effective jump detection threshold differs if costmap update interval is not constant.

Reply:
Moved in `spin()`. It will be called at 100hz.